src/mmw/sass/components/_layerpicker.scss

```diff
@@ -175,52 +175,10 @@ span.time-slider-end {
     width: 14px;
     height: 14px;
 
-    &.-park-federal {
-      background-color: #AFB42B;
-    }
-
-    &.-park-state {
-      background-color: #C0CA33;
-    }
-
-    &.-park-local {
-      background-color: #CDDC39;
-    }
-
-    &.-park-private {
-      background-color: #D4E157;
-    }
-
-    &.-park-unknown {
-      background-color: #E6EE9C;
-    }
-
-    &.-natural-resource-federal {
-      background-color: #388E3C;
-    }
-
-    &.-natural-resource-state {
-      background-color: #43A047;
-    }
-
-    &.-natural-resource-local {
-      background-color: #4CAF50;
-    }
-
-    &.-natural-resource-private {
-      background-color: #66BB6A;
-    }
-
-    &.-natural-resource-unknown {
-      background-color: #A5D6A7;
-    }
-
-    &.-conservation-easement {
-      background-color: #9E9446; // Originally #827717, changed for distinguishability
```

Comment:
@rajadain I updated the color ramp which satisfies https://github.com/WikiWatershed/model-my-watershed/pull/3228#issuecomment-575376563 . However this line has a comment that actually changes the conseration easement color in the legend from the value in the issue and that you gave - looks like that happened in this commit https://github.com/WikiWatershed/model-my-watershed/commit/dd1301c266494cf2ce65cc6c57c772b30c148747. Should I keep it as `#9E9446` for the chart and legend or should I use the `#827717` value?

Reply:
Ah good catch. Yes we should keep it at `#9E9446`. Currently it is `#A1887F` which is incorrect.